admin-dev/themes/new-theme/js/pages/product/edit/combinations-grid-renderer.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+import ProductMap from '@pages/product/product-map';
+import Router from '@components/router';
+
+const {$} = window;
+
+export default class CombinationsGridRenderer {
+  constructor() {
+    this.router = new Router();
+    this.eventEmitter = window.prestashop.instance.eventEmitter;
+    this.$combinationsTable = $(ProductMap.combinations.combinationsTable);
+    this.$combinationsTableBody = $(ProductMap.combinations.combinationsTableBody);
+    this.prototypeTemplate = this.$combinationsTable.data('prototype');
+    this.prototypeName = this.$combinationsTable.data('prototypeName');
+  }
+
+  render(data) {
+    this.renderCombinations(data.combinations);
+  }
+
+  // @todo: handle on change of new inputs of price quantity and is_default when they are ready
+  // @todo: they should call related endpoints to update combination
+  renderCombinations(combinations) {
+    this.$combinationsTableBody.empty();
+
+    let rowIndex = 0;
+    combinations.forEach((combination) => {
+      const row = this.getPrototypeRow(rowIndex);
+      this.$combinationsTableBody.append(row);
+
+      // fill inputs
+      const $combinationIdCell = $(ProductMap.combinations.tableRow.combinationIdCell(rowIndex));
```

Comment:
```suggestion
      const $combinationIdInput = $(ProductMap.combinations.tableRow.combinationIdInput(rowIndex));
```

This naming is confusing, I expected it to target the `td` but it's targeting the hidden input Which is fine but the naming should reflect this so it's easier to understand